lib/cloudflare/custom_hostnames.rb

```diff
@@ -9,79 +9,79 @@
 
 module Cloudflare
 
-  class CustomHostname < Representation
```

Comment:
This file only has spaces -> tabs changes

Reply:
Thanks for being consistent.